inductiva/storage/storage.py

```diff
@@ -256,27 +256,23 @@ def _list_files(root_path: str) -> Tuple[List[str], int]:
     return file_paths, total_size
 
 
-def remove_workspace(remote_dir, file_name: Optional[str] = None) -> bool:
+def remove_workspace(remote_dir) -> bool:
     """
-    Removes the specified file from a remote directory, or deletes the entire
-    remote directory if no file name is provided.
+    Removes path from a remote directory.
 
     Parameters:
     - remote_dir (str): The path to the remote directory.
-    - file_name (str, optional): The name of the file to remove within the
-        remote_dir. If omitted, the function removes the entire remote_dir.
     """
     api = storage_api.StorageApi(inductiva.api.get_client())
 
     logging.info("Removing workspace file(s)...")
-    if not file_name:
-        # make sure remote_dir ends with a '/'
-        remote_path = os.path.join(remote_dir, "")
-    else:
-        remote_path = os.path.join(remote_dir, file_name)
 
+    # Since we don't allow root files in workspaces it must be a directory
+    # otherwise path validation in the backend will give error
+    if '/' not in remote_dir:
+        remote_dir = remote_dir + '/'
     try:
-        api.delete_file(query_params={"path": remote_path},)
+        api.delete_file(query_params={"path": remote_dir},)
         logging.info("Workspace file(s) removed successfully.")
     except exceptions.ApiException as e:
```

Comment:
Why do we just except just to raise it ?

Reply:
Good point, I removed on other functions as well, can you confirm this is okay?